taca/server_status/server_status.py

```diff
@@ -107,3 +109,26 @@ def update_status_db(data, server_type=None):
             raise
         else:
             logging.info('{}: Server status has been updated'.format(key))
+
+def check_promethion_status():
+    config = CONFIG.get('promethion_status')
+    server = config.get('server')
+    path = config.get('path')
+    command = config.get('command')
+    command_to_run = f'{command} {path}'
+    user = config.get('user')
+
+    try:
+        subprocess.run(['ssh', '-t', f'{user}@{server}', command_to_run], 
+                       check=True)
+    except subprocess.CalledProcessError:
+        _send_promethion_warning_email()
+        return False
+    return True
+
+def _send_promethion_warning_email():
+    email_recipients = CONFIG.get('mail').get('recipients')
+    email_subject = ('An issue with the PromethION has been detected.')
+    email_message = ('An issue with the PromethION has been detected. '
```

Comment:
Maybe add a timestamp?

Reply:
Good point. It will be run as a cronjob so if there is an issue an email will be sent, so the time will be logged in the received email, wouldn't that be enough?

Reply:
Ah that's true enough.